src/ClusterTSnePlot.js

```diff
@@ -209,4 +216,8 @@ ClusterTSnePlot.propTypes = {
   tooltipContent: PropTypes.func
 }
 
+ClusterTSnePlot.defaultProps = {
+  showControls: true
```

Comment:
As you always pass the `showControls` from `TSnePlotView` component, you just need to clarify once in `TSnePlotView`, but do not need to specify it in `ClusterTSnePlot.defaultProps`. Same comment for `GeneExpressionTSnePlot`.

Reply:
Default prop was added in ClusterTSnePlot and GeneExpressionTSnePlot for the test cases respectively. Otherwise I have to specify showControl explicitly in each of them which would lead to more code change as compared to introducing defaultProp.

As in the test cases, we are testing ClusterTSnePlot and GeneExpressionTSnePlot and not TSnePlotView so can't get default value from there.